<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1121</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>assets/add-user.png</normaloff>assets/add-user.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="list">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>assets/kayit2.png</normaloff>assets/kayit2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>assets/start.png</normaloff>assets/start.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>591</width>
      <height>131</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 255, 75);
background-color: rgb(70, 255, 138);
color:rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>System is safe</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="stateTable">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>311</width>
     <height>651</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>date</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="videoset">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>721</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>assets/stop.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
